covid19-etl/etl/ncbi.py

```diff
@@ -627,19 +671,16 @@ async def _parse_big_query_response(self, response):
                     filesize,
                     file_name,
                     _,
-                ) = await self.file_helper.async_find_by_name(
-                    filename=virus_sequence["file_name"]
-                )
+                ) = await self.file_helper.async_find_by_name(filename=filename)
                 retrying = False
             except Exception as e:
                 print(
                     f"ERROR: Fail to get indexd for {filename}. Detail {e}. Retrying ..."
                 )
                 await asyncio.sleep(5)
 
-        assert (
-            did
-        ), f"file {filename} does not exist in the index, rerun NCBI_MANIFEST ETL"
+        if not did:
+            return False
```

Comment:
should we exit if `success==False` on line 261?

Reply:
No, we still want to submit the clinical data to virus_sequence_run_taxonomy node.  We don't just add a link to virus_sequence node